javascript-arrays/main.js

```diff
@@ -0,0 +1,30 @@
+const colors = ['red', 'white', 'blue'];
+
+console.log('Value of colors[0] ==> ', colors[0]);
+console.log('Value of colors[1] ==> ', colors[1]);
+console.log('Value of colors[2] ==> ', colors[2]);
+
+const str = 'America is ' + colors[0] + ' ' + colors[1] + ' and ' + colors[2] + '.';
+
+console.log(str);
+
+colors[2] = 'green';
+const str1 = 'Mexico is ' + colors[0] + ' ' + colors[1] + ' and ' + colors[2] + '.';
+
+console.log(str1);
+console.log('Value of array colors ==>', colors);
+
+const students = ['Joe', 'Donald', 'Barak', 'George'];
+const numberOfStudents = students.length;
+const str2 = ' There are 4 students in the class.' + ' \n The last student in the array is ' + students[3];
```

Comment:
Compare this string to the string in our example output in the instructions. The sentence doesn't match exactly. You should also be using your `numberOfStudents` variable rather than `students[3]`